sdk/tests/tutorials/ibor/test_cut_labels.py

```diff
@@ -0,0 +1,231 @@
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        cls.portfolios = lusid.PortfoliosApi(api_client)
+
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+        cls.cut_labels = lusid.CutLabelDefinitionsApi(api_client)
+
+    def test_cut_labels(self):
+        
+        def get_guid():
+            return str(uuid.uuid4())[:4]
+
+        # define function to format cut labels
+        def cut_label_formatter(date, cut_label_code):
+            return str(date) + "N" + cut_label_code
+        
+        
+        ## Create cut labels
+        # create a dictionary to keep track of cut labels
+        code = {}
+
+        def create_cut_label(hours, minutes, display_name, description, time_zone):
+            # Create the time for the cut label
+            time = models.CutLocalTime(hours=hours, minutes=minutes)
+            
+            # Define the parameters of the cut label in a request
+            request = models.CutLabelDefinition(
+                code=display_name+"-"+get_guid(), 
+                description=description, 
+                display_name=display_name,
+                cut_local_time=time,
+                time_zone=time_zone)
+
+            # Add the codes of our cut labels to our dictionary
+            code[request.display_name] = request.code 
+            
+            # Send the request to LUSID to create the cut label
+            result = self.cut_labels.create_cut_label_definition(
+                create_request=request)
+
+            # Check that result gives same details as input
+            self.assertEqual(result.display_name, display_name) # should this be self?
+            self.assertEqual(result.description, description)
+            self.assertEqual(result.cut_local_time, time)
+            self.assertEqual(result.time_zone, time_zone)
+
+        # Create cut labels for different time zones
+        create_cut_label(hours=9, minutes=0, display_name="LondonOpen", description="London Opening Time, 9am in UK", time_zone="GB")
+        create_cut_label(hours=17, minutes=0, display_name="LondonClose", description="London Closing Time, 5pm in UK", time_zone="GB")
+        create_cut_label(hours=9, minutes=0, display_name="SingaporeOpen", description="", time_zone="Singapore")
+        create_cut_label(hours=17, minutes=0, display_name="SingaporeClose", description="", time_zone="Singapore")
+        create_cut_label(hours=9, minutes=0, display_name="NYOpen", description="", time_zone="America/New_York")
+        create_cut_label(hours=17, minutes=0, display_name="NYClose", description="", time_zone="America/New_York")
+
+
+        ## Create portfolio
+        scope = 'cut_labels_demo'
+        portfolio_code = self.test_data_utilities.create_transaction_portfolio(scope)
+
+
+        ## Add instruments
```

Comment:
Perhaps "Get the instrument identifiers" or "Get the instruments" makes more sense here, this comment makes it sounds like you are adding them to LUSID.

Reply:
Done